### DISCUSSION TITLE
HTML on description of a Checkbox List

### DISCUSSION THREAD

**Discussion by nessimabadi** [filamentphp/filament#8192] [Help]
Hi,

Is there a way to have HTML on the description of a Checkbox List.

I see on the docs that in Helpers and Hints you can return an Htmlable element.

**Comment by nessimabadi** [comment]
Changing the return hint of getDescription in https://github.com/filamentphp/filament/blob/3.x/packages/forms/src/Components/CheckboxList.php#L309

from `: ?string` to `: string | Htmlable | null` seems to work

**Reply by danharrin** [reply]
You can make a PR if you want

**Reply by nessimabadi** [reply]
https://github.com/filamentphp/filament/pull/8194

Please do let me know if any changes are needed

Thanks for the amazin package
